docs/src/components/Typography/font-face.scss

```diff
@@ -26,8 +28,8 @@
 
   &__text {
     width: max-content;
-    font-size: $bpk-spacing-base * 2;
-    line-height: $bpk-spacing-base * 2;
+    font-size: $size;
+    line-height: $size;
```

Comment:
Was this producing a warning/error before? Or did you just refactor for fun?

Reply:
Yeah it was producing a weird stylelint warning